dags/main_summary.py

```diff
@@ -355,6 +355,8 @@
         cluster_by=["sample_id"],
         rename={"submission_date_s3": "submission_date"},
         replace=["SAFE_CAST(sample_id AS INT64) AS sample_id"],
+        env_vars={"GOOGLE_CLOUD_PROJECT":
+                  "{{ var.value.gcp_shared_prod_project }}"},
```

Comment:
Should we be using `{{ var.value.gcp_shared_prod_project }}` rather than `moz-fx-data-shared-prod` everywhere in the DAGs?

Reply:
Maybe. I think it might be a good pattern to adopt if we ever decide to make use of our stage environment for dag testing. It might be useful for local testing too.

I don't have a real strong opinion here and I could just change this to conform to what we have been doing.

Reply:
+1 for using `{{ var.value.gcp_shared_prod_project }}`